src/interface/can_translation/src/FloatReporterNode.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+ * Package:   can_translation
+ * Filename:  src/FloatReporterNode.cpp
+ * Author:    Joshua Williams
+ * Email:     joshmackwilliams@protonmail.com
+ * Copyright: 2021, Nova UTD
+ * License:   MIT License
+ */
+
+#include <string>
+#include <stdexcept>
+
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/float32.hpp"
+
+#include "voltron_msgs/msg/can_frame.hpp"
+#include "can_translation/FloatReporterNode.hpp"
+
+using namespace navigator::can_translation;
+
+FloatReporterNode::FloatReporterNode() : rclcpp::Node("float_reporter") {
+  this->declare_parameter("input_min");
+  this->params.input_min = this->get_parameter("input_min").as_int();
+  this->declare_parameter("input_max");
+  this->params.input_max = this->get_parameter("input_max").as_int();
+  this->declare_parameter("output_min");
+  this->params.output_min = this->get_parameter("output_min").as_double();
+  this->declare_parameter("output_max");
+  this->params.output_max = this->get_parameter("output_max").as_double();
+  this->declare_parameter("message_id");
+  this->params.message_id = this->get_parameter("message_id").as_int();
+  this->declare_parameter("field_start_bit");
+  this->params.field_start_bit = this->get_parameter("field_start_bit").as_int();
```

Comment:
You can also do 

`this->get_parameter("field_start_bit", this->params.field_start_bit);`, 

which loads it in directly through a reference. Personal preference, either way works 👍

Reply:
That's handy. I didn't know it worked that way. I will keep that in mind for next time.